Choosing a color

iAlgebra opened the pull request:
A color from the array of six colors was chosen and displayed as text on the title. This is the color that has been picked to be found.

iAlgebra committed:
Choosing a color

A color from the array of six colors was chosen and displayed as text on the title. This is the color that has been picked to be found.

fsodano commented on colorGame.js:
```diff
@@ -8,6 +8,10 @@ var colors = [
 ];
 
 var $squares = document.querySelectorAll(".square");
+var pickedColor = colors[3];
```
3 is a magical number, you should put it in a constant that describes what does it represent.

fsodano reviewed (changes requested):
Just a minor comment

iAlgebra committed:
Adding a constant and some comments

A specific colour was selected from the arrays of colours, its index is represented with the constant.
Some comments to clarify were added.
